### ISSUE TITLE
acme-default-renewal service occasionally freezes

### ISSUE THREAD

**Issue by alastairong** [Holo-Host/holo-nixpkgs#457]
One of the most common issues we seem to experience during registration is that HoloPorts register on the network but don't get issued a TLS cert. This is easily resolved by a reboot but often results in a customer service ticket.

**Comment by alastairong** [comment]
Diagnosis from @filalex77 

Some more research results:
- `simp_le` is almost unmaintained, doesn't support ACME v2 (which Let's Encrypt strongly advise to use instead of v1, which we are using)
- `lego` support is stable in 20.03, so if we can upgrade, this issue will be easier to work on, as new documentation is only for lego-related configuration (and simp_le is not used anymore in 20.03)
- we have custom logic (bash script) for provisioning the certificate with simp_le only after resolver responds for ${base36_id}.holohost.net; this can better be expressed by a required service

TL;DR: I assume our certificate logic is unmaintained -> broken. Before taking action, exact errors should be found (means debugging time). It would be much better to investigate after bump to 20.03.

**Comment by alastairong** [comment]
Closing as lets encrypt now using ACME v2
